atm.js

```diff
@@ -1,3 +1,219 @@
-$(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+ /*
+An application to keep track of the checking and savings balances.
+
+User actions
+ - As a user, I want to deposit money into one of the bank accounts
+ - As a user, I want to withdraw money from one of the bank accounts
+
+Validation / Overdraft protection
+   + If a withdrawal can be covered by the balances in both accounts, take the balance of the account withdrawn from down to $0 and take the rest of the withdrawal from the other account.
+   + If the withdrawal amount is more than the combined account balance, ignore it.
+
+Presentation
+ - The background of account to reflect it's balance.
+  */
+
+
+/**
+ * A class that represents user account.
+ */
+function Account() {
+
+  // State
+  this.balances = {
+    checking: 0,
+    savings: 0
+  };
+
+  /**
+   * Processes the deposit for the specified account type and amount.
+   * @param  {[String]} type   either "checking" or "savings"
+   * @param  {[String]} amount a numerical string
+   * @return true if the transaction was successful, else false.
```

Comment:
At first, I learned Java's convention when I was learning Android last year. https://en.wikipedia.org/wiki/Javadoc
I am trying to make sure that everything is readable for my future self.